### DISCUSSION TITLE
How to use Avalonia with PascalABC.NET?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#3007] [General]
There are too many nuget packages, I don't know which of them should be installed. I also need an example of hello world level, but I can't find anything like that. The samples are too advanced and too complex to me to translate to PascalABC.NET.

**Comment by SunSerega** [comment]
I don't think I'll be able to help you on this one. To make any good example, I would first need to dive quite deep myself, I'd estimate ~2 weeks to get comfortable. I don't have this time rn...

**Comment by sjsepan3** [comment]
> There are too many nuget packages, I don't know which of them should be installed. I also need an example of hello world level, but I can't find anything like that. The samples are too advanced and too complex to me to translate to PascalABC.NET.

This C#.Net app is a little bit more than a 'Hello World' level, but it's demonstrating menu, toolbar, and status bar.
[https://gitlab.com/sjsepan/AvaloniaApp1](https://gitlab.com/sjsepan/AvaloniaApp1)
